### DISCUSSION TITLE
How can we donate?

### DISCUSSION THREAD

**Discussion by eyadmba** [agronholm/apscheduler#741] [General]
I can clearly see how much effort is being put into this tool, and I know how tough it is to maintain open source code, so I was looking around for a way to donate but I couldn't find any. Is there a way to give donations? If there is, it's not easy to find...

**Comment by agronholm** [comment]
I thank you for the sentiment, but in my country, collecting donations requires a permit that costs 300 € and takes several months to process, so I've opted not to do so.
